language/google/cloud/language/entity.py

```diff
@@ -84,7 +80,7 @@ class Entity(object):
     def __init__(self, name, entity_type, metadata, salience, mentions):
         self.name = name
         self.entity_type = entity_type
-        self.wikipedia_url = metadata.pop('wikipedia_url', None)
+        self.wikipedia_url = metadata.get('wikipedia_url')
```

Comment:
Since you aren't `pop`-ing, ISTM that `self.wikipedia_url` should just be removed